src/modules/formBuilder/components/FormEditorItemPreview.tsx

```diff
@@ -0,0 +1,50 @@
+import { Typography } from '@mui/material';
+import React, { useState } from 'react';
+import CommonToggle from '@/components/elements/CommonToggle';
+import {
+  PreviewMode,
+  toggleItems,
+} from '@/modules/admin/components/forms/FormPreview';
+import DynamicField from '@/modules/form/components/DynamicField';
+import DynamicViewField from '@/modules/form/components/viewable/DynamicViewField';
+import { FormItem, ItemType } from '@/types/gqlTypes';
+
+interface FormEditorItemPreviewProps {
+  selectedItem: FormItem;
+  value?: any;
+  setValue: (value: any) => void;
+}
+
+const FormEditorItemPreview: React.FC<FormEditorItemPreviewProps> = ({
+  selectedItem,
+  value,
+  setValue,
+}) => {
+  const [toggleValue, setToggleValue] = useState<PreviewMode>('input');
+
+  if (selectedItem.type === ItemType.Group) return;
```

Comment:
This feels like it should go in the parent component. We don't want to render the Group based on context, because it doesn't fit in the layout. That's easier to understand than "this component always renders null for groups". Like `selectedItem.type !== ItemType.Group &&  <FormEditorItemPreview />`